pwa/src/components/objectEntities/ObjectEntityForm.tsx

```diff
@@ -22,30 +22,28 @@ import FlashMessage from 'react-flash-message';
 import LoadingOverlay from "../loadingOverlay/loadingOverlay";
```

Comment:
We're missing the `documentation` `const` declaration in this file:

```
const [documentation, setDocumentation] = React.useState<string>(null);
```

Also, the `handleSetDocumentation()` is never called so it doesn't run. Could be added underneath `handleSetApplications()` on line `41`.